### PR TITLE
[chore]: Update versions

### PR DESCRIPTION
**Pull request by Chaostheorie**

### CODE REVIEW

**Comment by Chaostheorie** [comment]
@wwwjfy This has to be tested and merged since ATM gino-starlette is not compatible with fastapi due to fastapi depending on starlette 0.16.

File: pyproject.toml [hunk 1]
```diff
@@ -22,20 +22,20 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
-starlette = "^0.13.0 || ^0.14.0"
+python = "^3.9"
```
Comment (wwwjfy) [inline pyproject.toml]: Is 3.9 necessary?

Comment (Chaostheorie) [reply pyproject.toml]: No, it isn't. Thank you for pointing it out, must've have overlooked the bump there. Just checked on starlette and Gino, and Python 3.6 should suffice.

**Comment by wwwjfy** [comment]
test failed. need to check.

**Comment by Chaostheorie** [comment]
Yep, I'll try to take a look at it this evening.

**Comment by Chaostheorie** [comment]
yada yada, some problem with 3.6 and a sub-dependency. Will most likely need to be attached to a new release since it's a breaking change.

**Comment by Chaostheorie** [comment]
@wwwjfy is everything fine now? Checks are passing.

**Comment by wwwjfy** [comment]
test still fails. seems related to https://github.com/pytest-dev/pytest/issues/7114 need to take another look

**Comment by wwwjfy** [comment]
The above test failed because poetry.lock was not updated. (need to run `poetry update` after updating pyproject.toml)

There was another test failing because of Starlette TestClient's adoption of a library anyio. I disabled that test for the moment and may revisit it later.

**Comment by Chaostheorie** [comment]
Thank you for the fix and merge.

**Pull request merged**